string/punctuation.js

```diff
@@ -50,6 +50,7 @@ const languageSpecificQuotes = {
 	en: ['“$1”', '‘$1’'], // English
 	fr: ['« $1 »', '‹ $1 ›'], // French
 	de: ['„$1“', '‚$1‘'], // German
+	he: ['”$1”', '’$’'], // Hebrew
```

Comment:
```suggestion
	he: ['”$1”', '’$1’'], // Hebrew
```

Reply:
A potential solution for the custom apostrophe works like this:
```suggestion
	he: ['”$1”', '’$1’', '׳'], // Hebrew
```
Additionally `const quotationRuleIndices = [0, 2, 5];` and the descriptions of both variables have to be adapted.